pheme/__init__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# pheme/__init__.py
```

Comment:
These lines would not be required. The `coding` is only necessary for Python 2. With Python 3 the encoding is always UTF 8 by default.